docs/workflows/oci-artifacts.rst

```diff
@@ -1,60 +0,0 @@
-Managing additional OCI media types
-===================================
-
-.. _default-oci-types:
-
-By default the following list of media types is enabled in the Container Registry::
```

Comment:
I would not remove entirely this file and leave a section in documentation stating that we are an OCI artifact registry. I would reword some of these docs with few examples what kind of additional to regular mediatypes we support ( out of the box) so it is clear for the end user that we are not just container but also artifact registry.

Reply:
How about we mention this information on our landing page (https://docs.pulpproject.org/pulp_container/)? I do not think it is necessary to have a dedicated page for it.

Reply:
@git-hyagi, do you think we should keep this doc section? If so, we may want to revisit the paragraphs and state clearly that we support `OCI Artifacts`, e.g., by referencing [helm charts](https://github.com/pulp/pulp_container/blob/55a5af36b4fa180a0df19d98479a010da903d7c3/docs/workflows/helm-support.rst) and providing more insights into what a generic OCI Artifact might look like. Of course, omitting the removed setting.

Reply:
I am in favor of keeping this doc section and rewriting the paragraphs to make it clear that this not just a container registry.